src/pages/documentation/mobile-core/signal/tabs/index.md

```diff
@@ -5,13 +5,13 @@
 Add the [Mobile Core](../index.md) extension to your project using the app's Gradle file.
 
 ```java
-implementation 'com.adobe.marketing.mobile:sdk-core:1.+'
+implementation 'com.adobe.marketing.mobile:sdk-core:2.+'
```

Comment:
Also since this is for Signal which has it's own version tag now, should we instead use
```
implementation "com.adobe.marketing.mobile:core:2.+"
implementation "com.adobe.marketing.mobile:signal:2.+"
```